client/telecomUniverseComponents/phone/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import 'ovh-api-services';
+
+import tucPhoneNumberDirective from './number/phone-number.directive';
+import tucPhoneNumberFilter from './number/phone-number.filter';
+
+export default angular
+  .module('tucPhone', [
+    'ovh-api-services',
+  ])
+  .directive('tucPhoneNumber', tucPhoneNumberDirective)
```

Comment:
You have to update this file too: https://github.com/ovh-ux/ovh-manager-telecom/blob/develop/client/app/telecom/telephony/line/calls/click2Call/telecom-telephony-line-calls-click2Call.html#L46